ui/shared/status/StatusChatInputReplyArea.qml

```diff
@@ -0,0 +1,100 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtGraphicalEffects 1.13
+import "../../imports"
+import "../../shared"
+
+Rectangle {
+    id: root
+    height: 50
+    color: Style.current.lightGrey
+    radius: 16
+    clip: true
+
+    property string userName: ""
+    property string message : ""
+    property string identicon: ""
+
+    signal closeButtonClicked()
+
+    Rectangle {
+        color: parent.color
+        anchors.bottom: parent.bottom
+        anchors.right: parent.right
+        height: parent.height / 2
+        width: 32
+        radius: 8
+    }
+
+    StyledText {
+        id: replyToUsername
+        text: "↪ " + userName
+        color: Style.current.black
+        anchors.top: parent.top
+        anchors.topMargin: 8
+        anchors.left: parent.left
+        anchors.leftMargin: Style.current.smallPadding
+        font.pixelSize: 13
+        font.weight: Font.Medium
+    }
+
+    StyledText {
+        id: replyText
+        text: Emoji.parse(message, "26x26")
+        anchors.left: replyToUsername.left
+        anchors.top: replyToUsername.bottom
+        anchors.topMargin: 2
+        anchors.right: parent.right
+        anchors.rightMargin: Style.current.padding
+        anchors.bottom: parent.bottom
+        elide: Text.ElideRight
+        font.pixelSize: 13
+        font.weight: Font.Normal
+        // Eliding only works for PlainText: https://bugreports.qt.io/browse/QTBUG-16567
+        textFormat: Text.PlainText
+        color: Style.current.black
+    }
+
+    RoundButton {
+        id: closeBtn
+        implicitWidth: 20
+        implicitHeight: 20
+        radius: 10
```

Comment:
10 sounds random. Usually we use 8 or 16. Any chance this was a mistake?

Reply:
The radius value really depends on what kind of radius you want. In case of generating a circle, you'll need a radius of the element's width - 50%, which in this case is `10`. `8` would be too little `16` would probably work, because you can't really go rounder than 10 in this case.

Reply:
Ah right, my bad, it makes sense. Usually I saw exactly that `width / 2` so it was obvious that it was round.